### ISSUE TITLE
Improve the bucket plotting by adding labels instead of bucket index

### ISSUE THREAD

**Issue by sbjelogr** [timvink/skorecard#88]
The current bucket plotting within the bucketers and the bucketing process, creates bars where the x-axis always indicates the bucket index.

It's probably more convenient to visualise the content of the bucket in the ticks (for example the edges of the bucket for numerical features or the content in case of categorical features)

**Comment by timvink** [comment]
Yup makes sense, and with the recent refactoring should be an edit to only one function.
